sections/main-product.liquid

```diff
@@ -1536,6 +1536,66 @@
       "default": false,
       "label": "t:sections.main-product.settings.enable_video_looping.label"
     },
+    {
+      "type": "header",
+      "content": "t:sections.main-product.settings.section_colors_heading"
```

Comment:
As mentioned earlier, if we add this heading under the `sections.all.colors` [here](https://github.com/Shopify/dawn/blob/f538b0a97903173c0298d8b268bcfb130664670a/locales/en.default.schema.json#L368-L386): then this would need to be updated to reflect that.